libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.kt

```diff
@@ -522,7 +540,7 @@ internal constructor(private val mapboxDirections: MapboxDirections) {
          * @return this builder for chaining options together
          * @since 0.26.0
          */
-        fun addWaypointTargets(vararg waypointTargets: Point): Builder {
+        fun addWaypointTargets(vararg waypointTargets: Point?): Builder {
```

Comment:
We need to revisit other `vararg` APIs (such as `waypointNames`, `approaches`, etc.) because Directions API accepts `null` and gets converted to `";"` so backend ignores it.

Reply:
get it, done